core/src/util/usFrameworkPrivate.cpp

```diff
@@ -168,6 +169,8 @@ void FrameworkPrivate::Shutdown(bool restart)
   case Bundle::STATE_STOPPING:
     // Shutdown already inprogress
     break;
+  default:
```

Comment:
The ```default``` case was meant as a run-time check to catch any conditions when the bundle state was not correct.

I can remove the ```default``` case if you believe it unnecessary. let me know.